models/comments-models.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+const checkExists = require("../utils/utils");
+
+exports.removeCommentById = (comment_id) => {
+  return Promise.all([
+    checkExists(
+      "comments",
+      "comment_id",
+      comment_id,
+      "Comment ID does not exist"
+    ),
```

Comment:
Is the Promise.all needed here? since checkExists is a promise we can just return that function.